logstash-input-kinesis.gemspec

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'logstash/inputs/kinesis/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "logstash-input-kinesis"
+  spec.version       = Logstash::Input::Kinesis::VERSION
+  spec.authors       = ["Brian Palmer"]
+  spec.email         = ["brian@codekitchen.net"]
+  spec.summary       = %q{Logstash plugin for Kinesis input}
+  spec.description   = %q{This gem is a logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/plugin install gemname. This gem is not a stand-alone program}
+  spec.homepage      = "https://github.com/codekitchen/logstash-input-kinesis"
+  spec.licenses      = ['Apache License (2.0)']
+
+  spec.files         = `git ls-files`.split($\) + Dir['vendor/jar-dependencies/runtime-jars/*.jar']
```

Comment:
Can you change this to:

```
s.files = Dir['lib/**/*','spec/**/*','vendor/**/*','*.gemspec','*.md','CONTRIBUTORS','Gemfile','LICENSE','NOTICE.TXT']
```

Using `git ls` was a hack for us, which didn't work well for offline installs.